src/frontend/data/sideEffects/getResourceList/index.ts

```diff
@@ -0,0 +1,53 @@
+import { stringify } from 'querystring';
+import { Dispatch } from 'redux';
+
+import { API_ENDPOINT, API_LIST_DEFAULT_PARAMS } from '../../../settings';
+import { requestStatus } from '../../../types/api';
+import { modelName } from '../../../types/models';
+import { addMultipleResources } from '../../genericReducers/resourceById/actions';
+import {
+  didGetResourceList,
+  failedToGetResourceList,
+  getResourceList as getResourceListAct,
+  ResourceListGet,
+} from '../../genericReducers/resourceList/actions';
+
+/**
+ * Makes and handles the GET request for a resource list.
+ * @param jwt The token to use to authenticate the request.
+ * @param resourceName The model name for the resource for which we're getting a list.
+ * @param params The parameters for the list request.
```

Comment:
You can update this docstring and explain why you are returning an async function ?